modules/shared/components/Select.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import { Select } from "@chakra-ui/react";
+
+interface Category {
```

Comment:
Podrias cambiarlo el nombre a `Props` o `SelectProps`

E importar los props del select de chakra para poder utilizarlos tambien en el componente

```suggestion
import { Select, SelectProps } from "@chakra-ui/react";

interface Props extends SelectProps {
```